packages/ee/billing/backend/usage.entity.ts

```diff
@@ -21,9 +21,9 @@ export const ProjectUsageEntity = new EntitySchema<ProjectUsageSchema>({
     },
     indices: [
         {
-            name: "idx_project_usage_project_id",
-            columns: ["projectId"],
-            unique: false,
+            name: "idx_project_usage_project_id_nextResetDatetime",
+            columns: ["projectId", "nextResetDatetime"],
+            unique: true,
```

Comment:
I think moving historic data to another entity altogether is safer to avoid doing complex queries when dealing with the current usage cycle